rrfs-test/validated_yamls/templates/obtype_config/aircar_airTemperature_133.yaml

```diff
@@ -1,9 +1,12 @@
      - obs space:
          name: aircar
+         distribution:
```

Comment:
line 2: 
`name: aircar`

Is `aircar` a keyword in JEDI? Does it affect JEDI functionalities if I change it to another name, such as `aircar_T_133`? 

@SamuelDegelia-NOAA @delippi @HuiLiu-NOAA @spanNOAA

Reply:
No, you can choose any name you like. From what I've learned, the name simply helps you quickly locate the related data process in the log file.

Reply:
> No, you can choose any name you like. From what I've learned, the name simply helps you quickly locate the related data process in the log file.

In this situation, let's change the obs space name to be unique. For this file, it should be `aircar_T_133`. Thanks!
@SamuelDegelia-NOAA

Reply:
@guoqing-noaa, this is another thing that I will be updating too. In my most recent [PR](https://github.com/NOAA-EMC/RDASApp/pull/188/files) I made this change for the msonet yamls . If is okay, I would like to make this change when I go through my "Phase 2" validation and update these yamls which will likely be completed in the next couple of weeks since I've already completed it for msonet. I don't think @SamuelDegelia-NOAA needs to update any of the yamls--I will handle that. Whatever the resultant "super.yaml" from the gen_yaml.sh is what we ought to use. These yamls are not the final product anyway plus the "super.yaml" will need to be updated many times as we continue to add more observations. I think it will work as is for now.

Reply:
@delippi unique `obs_space.name` is important as I will use it to filter different `observers`. This  change will not involve any tech/science issues and I think we better to get this updated as earlier as possible.

Reply:
@guoqing-noaa, I understand it is helpful for use via your yaml parsing tools, but I don't think this is the PR for that. Sam is just working on getting the ctests working. Please make these comments on my PRs for the yamls. Why do we need to use the yaml-tools to parse this anyway? We can just look at the modular yamls that this is built from in `rrfs-test/validated_yamls/templates/obtype_config/`.